testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -303,6 +304,21 @@ def test_chainid_quick_select():
         assert len(u.C.atoms) == 5
         assert len(u.D.atoms) == 7
 
+class TestTransformations(object):
+    """Tests the transformations keyword
+    """
+    def test_callable(self):
+        u = mda.Universe(PSF,DCD, transformations=translate([10,10,10]))
+        uref = mda.Universe(PSF,DCD)
+        ref = translate([10,10,10])(uref.trajectory[0])
+        assert_almost_equal(u.trajectory[0].positions, ref, decimal=6)
```

Comment:
So by calling `u.trajectory[0]` you trigger the reload of the first frame (ie it goes through `Reader.__getitem__`).  So currently it's not tested that doing `Universe(stuff, transformations=[stuff])` applies the transformations to the Universe on creation.  You should check this as `u.trajectory.ts.positions`, ref